neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
It is running inside bedrock. Running `brc slitaz brw lsb_release` gives me `fedora (implicit)`. Slitaz by default doesn't come with the `lsb_release` command or `/etc/os-release`